datastore/builtin/build_test.go

```diff
@@ -0,0 +1,207 @@
+package builtin
+
+import (
+	"github.com/drone/drone/common"
+	"github.com/franela/goblin"
+	"testing"
+)
+
+func TestBuildstore(t *testing.T) {
+	db := mustConnectTest()
+	rs := NewRepostore(db)
+	bs := NewBuildstore(db)
+	cs := NewCommitstore(db)
+	defer db.Close()
+
+	g := goblin.Goblin(t)
+	g.Describe("Buildstore", func() {
+
+		// before each test we purge the package table data from the database.
+		g.BeforeEach(func() {
+			db.Exec("DELETE FROM blobs")
+			db.Exec("DELETE FROM builds")
+			db.Exec("DELETE FROM commits")
+			db.Exec("DELETE FROM repos")
+			db.Exec("DELETE FROM stars")
+			db.Exec("DELETE FROM tasks")
+			db.Exec("DELETE FROM tokens")
+			db.Exec("DELETE FROM users")
+		})
+
+		g.It("NewBuildstore()", func() {
+			repo := common.Repo{
+				UserID: 1,
+				Owner:  "oliveiradan",
+				Name:   "drone-test1",
+			}
+			//Add repo
+			_err1 := rs.AddRepo(&repo)
+			_err2 := rs.SetRepo(&repo)
+			getrepo, _err3 := rs.Repo(repo.ID)
+			g.Assert(_err1 == nil).IsTrue()
+			g.Assert(_err2 == nil).IsTrue()
+			g.Assert(_err3 == nil).IsTrue()
+			g.Assert(repo.ID).Equal(getrepo.ID)
+
+			//Add build
+			build := common.Build{
+				ID:       1,
+				CommitID: 1,
+				State:    "success",
+			}
+			_err1 = bs.SetBuild(&build)
```

Comment:
only use underscores when the variable name conflicts with another variable or a package. For example, if I import the `url` package I cannot have a variable called `url` since they will  clash, so I might choose to name it `url_`

so generally speaking there should be very few veriables that are suffixed with an underscore